src/pages/manage_dac/ManageDac.js

```diff
@@ -189,86 +190,106 @@ class AdminManageDac extends Component {
+            className: 'col-md-6 col-xs-12 search-wrapper no-padding'}, [
+            div({className: 'col-xs-6'}, [
+              h(SearchBox, {
+                id: 'manageDac',
+                searchHandler: this.handleSearchDac,
+                pageHandler: this.handlePageChange,
+                color: 'common'
+              })
             ]),
-
-            a({
-              id: 'btn_addDAC',
-              className: 'col-lg-6 col-md-6 col-sm-6 col-xs-6 btn-primary btn-add common-background',
-              onClick: this.addDac
-            }, [
-              div({ className: 'all-icons add-dac_white' }),
-              span({}, ['Add Data Access Committee'])
+              a({
+                id: 'btn_addDAC',
+                className: 'col-xs-6 btn-primary btn-add common-background',
+                onClick: this.addDac
+              }, [
+                div({className: 'all-icons add-dac_white'}),
+                span({}, ['Add Data Access Committee'])
+            ])
+          ]),
+          div({
+            isRendered: (this.props.match.params.role === "chair"),
+            className: 'search-wrapper no-padding'}, [
+            div({className: 'col-xs-6'}, [
+              h(SearchBox, {
+                id: 'manageDac',
+                searchHandler: this.handleSearchDac,
+                pageHandler: this.handlePageChange,
+                color: 'common'
+              })
             ])
           ])
         ]),
-        div({ className: 'jumbotron table-box' }, [
-          div({ className: 'grid-9-row' }, [
-            div({ className: 'col-2 cell-header cell-sort common-color',
-              onClick: this.sort('name', !this.state.descendingOrder)}, [
+        div({className: 'jumbotron table-box'}, [
+          div({className: 'grid-9-row'}, [
+            div({
+              className: 'col-2 cell-header cell-sort common-color',
+              onClick: this.sort('name', !this.state.descendingOrder)
+            }, [
               'DAC Name',
-              span({ className: 'glyphicon sort-icon glyphicon-sort' })]),
-            div({ className: 'col-3 cell-header common-color' }, ['DAC Description']),
-            div({ className: 'col-2 cell-header common-color' }, ['DAC Datasets']),
-            div({ className: 'col-2 cell-header f-center common-color' }, ['Actions'])
+              span({className: 'glyphicon sort-icon glyphicon-sort'})]),
+            div({className: 'col-3 cell-header common-color'}, ['DAC Description']),
+            div({className: 'col-2 cell-header common-color'}, ['DAC Datasets']),
+            div({className: 'col-2 cell-header f-center common-color'}, ['Actions'])
           ]),
 
-          hr({ className: 'table-head-separator' }),
+          hr({className: 'table-head-separator'}),
 
           this.state.dacs.filter(this.searchTable(searchDacText))
             .slice((currentPage - 1) * limit, currentPage * this.state.limit)
             .map(dac => {
```

Comment:
In the case of a chairperson on a single dac, this map shows me ALL DACs, not the ones that I'm a chair in. That's because all DACs are visible to all user roles. We need to filter the DACs on this page to the ones the user is a chairperson in, or all of them if they're an admin.